transforms/tabular/tabular-merger-plugin/README.md

```diff
@@ -0,0 +1,53 @@
+# Tabular Merger (v0.1.0)
+
+This WIPP plugin merges all tabular files with vaex supported file formats into a combined file using either row or column merging.
+```
+1. csv
+2. hdf5
+3. parquet
+4. feather
+5. arrow
+```
```

Comment:
You don't need the triple ticks around lists, just an empty line on the top and bottom.

Reply:
done